framework-cloud/framework-openshift/pom.xml

```diff
@@ -57,6 +57,11 @@
       <artifactId>commons-io</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
```

Comment:
Personally, I'd say that including the entire library just to use a simple `StringUtils.isEmpty()` is overkill.

Reply:
I agree. I wanted to use it for something else, but in the end I didn't. Removed